src/stores/auth.ts

```
@@ -1,78 +1,68 @@
+    document.location.replace('/login')
+  }
+
+  async function Login(email: string, password: string) {
+    const response = await login({ email, password });
+    SetTokens(response.accessToken, response.refreshToken);
+    return response;
+  }
+
+  async function GetLoggedInUser() {
+    const user = await getLoggedInUser();
+    User.value = user!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)